typhon/cli.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import logging
+import sys
+
+import coloredlogs
+from qtpy.QtWidgets import QApplication
+
+import typhon
+
+logger = logging.getLogger(__name__)
+
+# Argument Parser Setup
+parser = argparse.ArgumentParser(description='Create a TyphonSuite for '
+                                             'device/s stored in a Happi '
+                                             'Database')
+
+parser.add_argument('devices', nargs='*',
+                    help='Device names to load in the TyphonSuite')
+parser.add_argument('--happi-cfg',
+                    help='Location of happi configuration file '
+                         'if not specified by $HAPPI_CFG environment variable')
+parser.add_argument('--version', action='store_true',
+                    help='Current version and location '
+                         'of Typhon installation.')
+parser.add_argument('--debug', action='store_true',
+                    help='Show the debug logging stream')
+parser.add_argument('--dark', action='store_true',
+                    help='Use the QDarkStyleSheet shipped with typhon')
+parser.add_argument('--stylesheet',
+                    help='Additional stylesheet options')
+
```

Comment:
One more cool thing you can do since you set up the cli like this is to include the `--help` text inside your sphinx docs, see https://github.com/pcdshub/hutch-python/blob/cde63c1b0c03adebe32741a2afb5fcba0557a83c/hutch_python/cli.py#L41 where my autodocs are picking up the module's `__doc__` string.

Reply:
This is a very good idea